roles/common/tasks/cisco_nxos.yaml

```diff
@@ -0,0 +1,78 @@
+---
+# Define nxos connection parameters
+- name: Define Connection Parameters
+  ansible.builtin.set_fact:
+    ansible_ssh_user: "{{ sw_secret['user'] }}"
+    ansible_ssh_pass: "{{ sw_secret['pass'] }}"
+
+- name: Enable sftp-server feature
+  cisco.nxos.nxos_feature:
+    feature: sftp-server
+    state: enabled
+  notify: Save Config
+
+- name: Enable scp-server feature
+  cisco.nxos.nxos_feature:
+    feature: scp-server
+    state: enabled
+  notify: Save Config
+
+- name: Enable vpc feature
+  cisco.nxos.nxos_feature:
+    feature: vpc
+    state: enabled
+  notify: Save Config
+
+- name: Enable lacp feature
```

Comment:
@hakasapl you probably want to enable the vlan interface feature as part of this playbook then.

Reply:
Yeah definitely